docs/internal-documentation/maintenance/ReleaseProcess-deb-packages.md

```diff
@@ -0,0 +1,175 @@
+Release Process - Create .deb Packages
+=====================================
+
+
+## Overview
+
+The purpose of this process is to create and publish .deb packages for
+Zonemaster components, currently only Zonemaster-CLI, Zonemaster-Engine and
+Zonemaster-LDNS are covered.
+
+The process to build the packages is split in two steps. In the first step "nightly
+packages" are updated. In the second step packages are promoted to "stable packages".
+
+## 1. Update the nightly packages
+
+Nightly packages are based on the develop branch of Zonemaster git repositories.
+The sources for nightly packages are kept in the branch `<distribution>-nightly`
+of the [Packages sources] repository.
+
+For each package:
+
+1. If dependencies have changed, update the `Build-Depends` and `Depends`
+   section of the  `debian/control` file for each  required package to add
+   and/or remove any dependencies that have changed.
+
+2. If there are new files (executables as such), update `debian/*.manpages` and
+   `debian/*.install` accordingly. This step is not required for new Perl
+   modules, but will be for new scripts.
+
+## 2. Promote the nightly build to stable
+
+
+When a new Zonemaster release of the relevant component has been
+[published on Github], and when the nightly package is working fine, promote it
+to stable. The only difference between a nightly packages and its promoted
+version is the upstream tarball. While the nightly packages use the develop
+branch as upstream, the stable ones use a specific tag.
+
+1. Locally merge the `<distribution>-nightly` branch into `<distribution>` of
+   the [Packages sources] repository. The `.gitlab-ci.yml` and `pkg.sh` files
+   should not be merged.
+
+2. Update the upstream ref in `pkg.sh` with the tag of the corresponding release
+   for each package.
+
+3. Update the `changelog` for each package, by adding a new entry. The first line
+   should contain the package version in format
+   `<project version>-<package version>+deb<debian version>`.
+
+   For example:  `zonemaster-cli (3.1.0-3+deb11)` is the partial entry for
+   Zonemaster CLI version `3.1.0`, the package version is `3` and it is built
+   for Debian 11. Package version is incremented when the package sources are
+   updated but the upstream version remains the same.
```

Comment:
"upstream version" is that the same as "<debian version>"? Please use the same term then, else explain.

Reply:
Please see the corrected comment (never put `<>` in `""`).

Reply:
upstream version -> zonemaster version
debian version -> os version

Reply:
> upstream version -> zonemaster component version debian version -> os version

Can you please update the document with the clarification?

Reply:
Done